game/state/rules/city/vehicletype.h

```diff
@@ -14,7 +14,6 @@
 
```

Comment:
Adding
#include "framework/logger.h"

here fixes the LogError() issues below for me.